static/lang/re-en.json

```diff
@@ -1979,8 +1979,8 @@
             },
             "Monk": {
                 "OneInchPunch": {
-                    "ThreeActionLabel": "One Inch Punch (Three Actions)",
-                    "TwoActionLabel": "One Inch Punch (Two Actions)"
```

Comment:
Problem with this change is it will suddenly look weird on actors with the old version of the feat. Also `One Action`, `Two Actions`, and `Three Actions` should be generic enough to go in a generic area (and probably exist already).

Reply:
There doesn't seem to be "One/Two/Three Actions" in en.json. Worth adding them?

Reply:
I think so, yeah. Add them in re-en.json under SpecificRule.